src/scss/layout/_header.scss

```diff
@@ -1,29 +1,48 @@
 @use '../../../node_modules/accoutrement/sass/tools';
 
 [data-layout='header'] {
-  --angle: var(--edge);
-  --edge: calc(var(--body-inline) / 2);
-  --overlap: var(--gutter);
-  margin: 0 calc(var(--body-inline) * -1) var(--spacer);
   display: grid;
   grid-area: header;
+}
+
+[data-header='hero'] {
+  --angle: var(--edge);
+  --edge: calc(var(--page-margin) / 2);
+  --overlap: var(--gutter);
+  --newline: var(--shim);
+  margin: 0 calc(var(--page-margin) * -1) var(--gutter);
   grid-template:
     'hero hero' auto
     'title hero2' var(--overlap)
     'title .' auto
     'meta meta' auto
+    'intro intro' auto
     / auto minmax(var(--edge), 1fr);
 
+  #title {
+    margin-bottom: 0;
+    padding-left: var(--page-margin);
+  }
+
+  #meta,
+  #intro {
+    padding: 0 var(--page-margin);
+  }
+
   @media (min-width: 35em) and (min-height: 35em) {
```

Comment:
should we use the new media-page variable here? I know we are using rems when it is declared though so maybe there is a difference?

Reply:
Mainly I was thinking that they aren't really tied together. We could change the size we want for a media-page, and there's no reason for these media-queries to change along with that. I'm also trying to be a bit more careful about what belongs as a global pattern, and what really is just a component one-off. These numbers aren't really tied to anything else - just what makes sense for the header layout.